EventFilter/CTPPSRawToDigi/interface/RawDataUnpacker.h

```diff
@@ -26,7 +27,14 @@ class RawDataUnpacker
     typedef uint64_t word;
 
     /// VFAT transmission modes
-    enum { vmCluster = 0x80, vmRaw = 0x90 };
+    enum { vmCluster = 0x80, vmRaw = 0x90, vmDiamondCompact = 0xB0 };
```

Comment:
Within ConfDB we can not have bla::foobar but only blafoobar as module plugin name.
So if this unpacker needs to run within HLT eventually, you'd need to make a unique name,
such as CTPPSRawDataUnpacker.